src/config/inputs.rs

```diff
@@ -40,17 +16,15 @@ pub(crate) struct ConfigInput {
 impl ConfigInput {
     /// Read config fragments and merge them into a single config.
     pub(crate) fn read_configs(
-        dirs: &[path::PathBuf],
+        dirs: &Vec<String>,
```

Comment:
* If consumer passes ownership to this: `Vec<String>`
* If consumer does not pass ownership to this: `&[String]`

Reply:
Thanks, fixed!